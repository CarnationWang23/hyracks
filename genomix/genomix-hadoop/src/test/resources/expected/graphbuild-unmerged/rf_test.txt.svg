<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
 "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<!-- Generated by graphviz version 2.26.3 (20100126.1600)
 -->
<!-- Title: rf_test_txt Pages: 1 -->
<svg width="516pt" height="428pt"
 viewBox="0.00 0.00 516.00 428.00" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="graph1" class="graph" transform="scale(1 1) rotate(0) translate(4 424)">
<title>rf_test_txt</title>
<polygon fill="white" stroke="white" points="-4,5 -4,-424 513,-424 513,5 -4,5"/>
<g id="graph2" class="cluster"><title>cluster_legend</title>
<polygon fill="none" stroke="black" points="49,-8 49,-209 211,-209 211,-8 49,-8"/>
<text text-anchor="middle" x="130" y="-192.4" font-family="Times Roman,serif" font-size="14.00">legend</text>
</g>
<g id="graph3" class="cluster"><title>cluster_1</title>
<polygon fill="none" stroke="black" points="8,-324 8,-412 500,-412 500,-324 8,-324"/>
</g>
<g id="graph4" class="cluster"><title>cluster_2</title>
<polygon fill="none" stroke="black" points="8,-217 8,-305 500,-305 500,-217 8,-217"/>
</g>
<!-- legend_0_0 -->
<g id="node2" class="node"><title>legend_0_0</title>
<ellipse fill="black" stroke="black" cx="59" cy="-157" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_0 -->
<g id="node3" class="node"><title>legend_1_0</title>
<ellipse fill="black" stroke="black" cx="201" cy="-157" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_0&#45;&gt;legend_1_0 -->
<g id="edge3" class="edge"><title>legend_0_0&#45;&gt;legend_1_0</title>
<path fill="none" stroke="#dd1e2f" d="M61.0074,-157C74.8673,-157 156.744,-157 188.46,-157"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="188.862,-160.5 198.862,-157 188.861,-153.5 188.862,-160.5"/>
<text text-anchor="middle" x="130" y="-162.4" font-family="Times Roman,serif" font-size="14.00">FF</text>
</g>
<!-- legend_0_1 -->
<g id="node5" class="node"><title>legend_0_1</title>
<ellipse fill="black" stroke="black" cx="59" cy="-116" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_1 -->
<g id="node6" class="node"><title>legend_1_1</title>
<ellipse fill="black" stroke="black" cx="201" cy="-116" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_1&#45;&gt;legend_1_1 -->
<g id="edge5" class="edge"><title>legend_0_1&#45;&gt;legend_1_1</title>
<path fill="none" stroke="#ebb035" d="M61.0074,-116C74.8673,-116 156.744,-116 188.46,-116"/>
<polygon fill="#ebb035" stroke="#ebb035" points="188.862,-119.5 198.862,-116 188.861,-112.5 188.862,-119.5"/>
<text text-anchor="middle" x="130" y="-121.4" font-family="Times Roman,serif" font-size="14.00">FR</text>
</g>
<!-- legend_0_2 -->
<g id="node8" class="node"><title>legend_0_2</title>
<ellipse fill="black" stroke="black" cx="59" cy="-75" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_2 -->
<g id="node9" class="node"><title>legend_1_2</title>
<ellipse fill="black" stroke="black" cx="201" cy="-75" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_2&#45;&gt;legend_1_2 -->
<g id="edge7" class="edge"><title>legend_0_2&#45;&gt;legend_1_2</title>
<path fill="none" stroke="#06a2cb" d="M61.0074,-75C74.8673,-75 156.744,-75 188.46,-75"/>
<polygon fill="#06a2cb" stroke="#06a2cb" points="188.862,-78.5001 198.862,-75 188.861,-71.5001 188.862,-78.5001"/>
<text text-anchor="middle" x="130" y="-80.4" font-family="Times Roman,serif" font-size="14.00">RF</text>
</g>
<!-- legend_0_3 -->
<g id="node11" class="node"><title>legend_0_3</title>
<ellipse fill="black" stroke="black" cx="59" cy="-34" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_3 -->
<g id="node12" class="node"><title>legend_1_3</title>
<ellipse fill="black" stroke="black" cx="201" cy="-34" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_3&#45;&gt;legend_1_3 -->
<g id="edge9" class="edge"><title>legend_0_3&#45;&gt;legend_1_3</title>
<path fill="none" stroke="#218559" d="M61.0074,-34C74.8673,-34 156.744,-34 188.46,-34"/>
<polygon fill="#218559" stroke="#218559" points="188.862,-37.5001 198.862,-34 188.861,-30.5001 188.862,-37.5001"/>
<text text-anchor="middle" x="130" y="-39.4" font-family="Times Roman,serif" font-size="14.00">RR</text>
</g>
<!-- 1,1 -->
<g id="node15" class="node"><title>1,1</title>
<ellipse fill="none" stroke="black" cx="59" cy="-368" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="39.5" y="-379.167" font-family="Times Roman,serif" font-size="10.00">1,1&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="37,-360 37,-374 82,-374 82,-360 37,-360"/>
<text text-anchor="start" x="42.5" y="-364.667" font-family="Times Roman,serif" font-size="10.00">ACGGT</text>
<polygon fill="#218559" stroke="#218559" points="37,-346 37,-360 82,-360 82,-346 37,-346"/>
<text text-anchor="start" x="43" y="-350.667" font-family="Times Roman,serif" font-size="10.00">ACCGT</text>
</g>
<!-- 1,2 -->
<g id="node16" class="node"><title>1,2</title>
<ellipse fill="none" stroke="black" cx="201" cy="-368" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="181.5" y="-379.167" font-family="Times Roman,serif" font-size="10.00">1,2&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="179,-360 179,-374 224,-374 224,-360 179,-360"/>
<text text-anchor="start" x="184" y="-364.667" font-family="Times Roman,serif" font-size="10.00">CGGTG</text>
<polygon fill="#218559" stroke="#218559" points="179,-346 179,-360 224,-360 224,-346 179,-346"/>
<text text-anchor="start" x="184" y="-350.667" font-family="Times Roman,serif" font-size="10.00">CACCG</text>
</g>
<!-- 1,1&#45;&gt;1,2 -->
<g id="edge29" class="edge"><title>1,1&#45;&gt;1,2</title>
<path fill="none" stroke="#dd1e2f" d="M101.605,-361.755C116.207,-361.208 132.729,-361.105 148.049,-361.448"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="148.326,-364.957 158.425,-361.756 148.534,-357.96 148.326,-364.957"/>
</g>
<!-- 2,2 -->
<g id="node21" class="node"><title>2,2</title>
<ellipse fill="none" stroke="black" cx="201" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="181.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">2,2&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="179,-253 179,-267 224,-267 224,-253 179,-253"/>
<text text-anchor="start" x="184.5" y="-257.667" font-family="Times Roman,serif" font-size="10.00">CCGTG</text>
<polygon fill="#218559" stroke="#218559" points="179,-239 179,-253 224,-253 224,-239 179,-239"/>
<text text-anchor="start" x="183.5" y="-243.667" font-family="Times Roman,serif" font-size="10.00">CACGG</text>
</g>
<!-- 1,1&#45;&gt;2,2 -->
<g id="edge31" class="edge"><title>1,1&#45;&gt;2,2</title>
<path fill="none" stroke="#06a2cb" d="M96.6544,-349.87C117.807,-339.685 140,-329 140,-329 140,-329 153.666,-313.765 168.062,-297.718"/>
<polygon fill="#06a2cb" stroke="#06a2cb" points="171.058,-299.619 175.131,-289.838 165.848,-294.945 171.058,-299.619"/>
</g>
<!-- 1,2&#45;&gt;1,1 -->
<g id="edge37" class="edge"><title>1,2&#45;&gt;1,1</title>
<path fill="none" stroke="#218559" d="M158.425,-374.244C143.825,-374.792 127.305,-374.895 111.982,-374.553"/>
<polygon fill="#218559" stroke="#218559" points="111.704,-371.043 101.605,-374.245 111.497,-378.04 111.704,-371.043"/>
</g>
<!-- 1,3 -->
<g id="node17" class="node"><title>1,3</title>
<ellipse fill="none" stroke="black" cx="325" cy="-368" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="305.5" y="-379.167" font-family="Times Roman,serif" font-size="10.00">1,3&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="303,-360 303,-374 348,-374 348,-360 303,-360"/>
<text text-anchor="start" x="309" y="-364.667" font-family="Times Roman,serif" font-size="10.00">GGTGT</text>
<polygon fill="#218559" stroke="#218559" points="303,-346 303,-360 348,-360 348,-346 303,-346"/>
<text text-anchor="start" x="308.5" y="-350.667" font-family="Times Roman,serif" font-size="10.00">ACACC</text>
</g>
<!-- 1,2&#45;&gt;1,3 -->
<g id="edge33" class="edge"><title>1,2&#45;&gt;1,3</title>
<path fill="none" stroke="#dd1e2f" d="M243.327,-361.562C252.601,-361.258 262.531,-361.176 272.159,-361.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="272.364,-364.822 282.443,-361.554 272.527,-357.823 272.364,-364.822"/>
</g>
<!-- 2,1 -->
<g id="node20" class="node"><title>2,1</title>
<ellipse fill="none" stroke="black" cx="59" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="39.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">2,1&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="37,-253 37,-267 82,-267 82,-253 37,-253"/>
<text text-anchor="start" x="43" y="-257.667" font-family="Times Roman,serif" font-size="10.00">ACCGT</text>
<polygon fill="#218559" stroke="#218559" points="37,-239 37,-253 82,-253 82,-239 37,-239"/>
<text text-anchor="start" x="42.5" y="-243.667" font-family="Times Roman,serif" font-size="10.00">ACGGT</text>
</g>
<!-- 1,2&#45;&gt;2,1 -->
<g id="edge35" class="edge"><title>1,2&#45;&gt;2,1</title>
<path fill="none" stroke="#06a2cb" d="M167.587,-345.312C145.274,-330.161 120,-313 120,-313 120,-313 109.378,-303.945 96.9983,-293.392"/>
<polygon fill="#06a2cb" stroke="#06a2cb" points="99.0233,-290.519 89.1426,-286.695 94.4822,-295.846 99.0233,-290.519"/>
</g>
<!-- 1,3&#45;&gt;1,2 -->
<g id="edge41" class="edge"><title>1,3&#45;&gt;1,2</title>
<path fill="none" stroke="#218559" d="M282.443,-374.446C273.158,-374.745 263.225,-374.824 253.602,-374.681"/>
<polygon fill="#218559" stroke="#218559" points="253.407,-371.175 243.327,-374.438 253.242,-378.173 253.407,-371.175"/>
</g>
<!-- 1,4 -->
<g id="node18" class="node"><title>1,4</title>
<ellipse fill="none" stroke="black" cx="449" cy="-368" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="429.5" y="-379.167" font-family="Times Roman,serif" font-size="10.00">1,4&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="427,-360 427,-374 472,-374 472,-360 427,-360"/>
<text text-anchor="start" x="434" y="-364.667" font-family="Times Roman,serif" font-size="10.00">GTGTA</text>
<polygon fill="#218559" stroke="#218559" points="427,-346 427,-360 472,-360 472,-346 427,-346"/>
<text text-anchor="start" x="433.5" y="-350.667" font-family="Times Roman,serif" font-size="10.00">TACAC</text>
</g>
<!-- 1,3&#45;&gt;1,4 -->
<g id="edge39" class="edge"><title>1,3&#45;&gt;1,4</title>
<path fill="none" stroke="#dd1e2f" d="M367.327,-361.562C376.601,-361.258 386.531,-361.176 396.159,-361.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="396.364,-364.822 406.443,-361.554 396.527,-357.823 396.364,-364.822"/>
</g>
<!-- 1,4&#45;&gt;1,3 -->
<g id="edge43" class="edge"><title>1,4&#45;&gt;1,3</title>
<path fill="none" stroke="#218559" d="M406.443,-374.446C397.158,-374.745 387.225,-374.824 377.602,-374.681"/>
<polygon fill="#218559" stroke="#218559" points="377.407,-371.175 367.327,-374.438 377.242,-378.173 377.407,-371.175"/>
</g>
<!-- 2,1&#45;&gt;1,2 -->
<g id="edge15" class="edge"><title>2,1&#45;&gt;1,2</title>
<path fill="none" stroke="#06a2cb" d="M92.4134,-283.688C114.726,-298.839 140,-316 140,-316 140,-316 150.622,-325.055 163.002,-335.608"/>
<polygon fill="#06a2cb" stroke="#06a2cb" points="160.977,-338.481 170.857,-342.305 165.518,-333.154 160.977,-338.481"/>
</g>
<!-- 2,1&#45;&gt;2,2 -->
<g id="edge13" class="edge"><title>2,1&#45;&gt;2,2</title>
<path fill="none" stroke="#dd1e2f" d="M101.605,-254.755C116.207,-254.208 132.729,-254.105 148.049,-254.448"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="148.326,-257.957 158.425,-254.756 148.534,-250.96 148.326,-257.957"/>
</g>
<!-- 2,2&#45;&gt;1,1 -->
<g id="edge19" class="edge"><title>2,2&#45;&gt;1,1</title>
<path fill="none" stroke="#06a2cb" d="M168.906,-285.184C148.325,-300.692 121.435,-320.954 99.4042,-337.555"/>
<polygon fill="#06a2cb" stroke="#06a2cb" points="97.1022,-334.907 91.222,-343.72 101.315,-340.497 97.1022,-334.907"/>
</g>
<!-- 2,2&#45;&gt;2,1 -->
<g id="edge21" class="edge"><title>2,2&#45;&gt;2,1</title>
<path fill="none" stroke="#218559" d="M158.425,-267.244C143.825,-267.792 127.305,-267.895 111.982,-267.553"/>
<polygon fill="#218559" stroke="#218559" points="111.704,-264.043 101.605,-267.245 111.497,-271.04 111.704,-264.043"/>
</g>
<!-- 2,3 -->
<g id="node22" class="node"><title>2,3</title>
<ellipse fill="none" stroke="black" cx="325" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="305.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">2,3&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="303,-253 303,-267 348,-267 348,-253 303,-253"/>
<text text-anchor="start" x="308" y="-257.667" font-family="Times Roman,serif" font-size="10.00">CGTGG</text>
<polygon fill="#218559" stroke="#218559" points="303,-239 303,-253 348,-253 348,-239 303,-239"/>
<text text-anchor="start" x="308" y="-243.667" font-family="Times Roman,serif" font-size="10.00">CCACG</text>
</g>
<!-- 2,2&#45;&gt;2,3 -->
<g id="edge17" class="edge"><title>2,2&#45;&gt;2,3</title>
<path fill="none" stroke="#dd1e2f" d="M243.327,-254.562C252.601,-254.258 262.531,-254.176 272.159,-254.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="272.364,-257.822 282.443,-254.554 272.527,-250.823 272.364,-257.822"/>
</g>
<!-- 2,3&#45;&gt;2,2 -->
<g id="edge25" class="edge"><title>2,3&#45;&gt;2,2</title>
<path fill="none" stroke="#218559" d="M282.443,-267.446C273.158,-267.745 263.225,-267.824 253.602,-267.681"/>
<polygon fill="#218559" stroke="#218559" points="253.407,-264.175 243.327,-267.438 253.242,-271.173 253.407,-264.175"/>
</g>
<!-- 2,4 -->
<g id="node23" class="node"><title>2,4</title>
<ellipse fill="none" stroke="black" cx="449" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="429.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">2,4&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="427,-253 427,-267 472,-267 472,-253 427,-253"/>
<text text-anchor="start" x="433" y="-257.667" font-family="Times Roman,serif" font-size="10.00">GTGGT</text>
<polygon fill="#218559" stroke="#218559" points="427,-239 427,-253 472,-253 472,-239 427,-239"/>
<text text-anchor="start" x="432.5" y="-243.667" font-family="Times Roman,serif" font-size="10.00">ACCAC</text>
</g>
<!-- 2,3&#45;&gt;2,4 -->
<g id="edge23" class="edge"><title>2,3&#45;&gt;2,4</title>
<path fill="none" stroke="#dd1e2f" d="M367.327,-254.562C376.601,-254.258 386.531,-254.176 396.159,-254.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="396.364,-257.822 406.443,-254.554 396.527,-250.823 396.364,-257.822"/>
</g>
<!-- 2,4&#45;&gt;2,3 -->
<g id="edge27" class="edge"><title>2,4&#45;&gt;2,3</title>
<path fill="none" stroke="#218559" d="M406.443,-267.446C397.158,-267.745 387.225,-267.824 377.602,-267.681"/>
<polygon fill="#218559" stroke="#218559" points="377.407,-264.175 367.327,-267.438 377.242,-271.173 377.407,-264.175"/>
</g>
</g>
</svg>
