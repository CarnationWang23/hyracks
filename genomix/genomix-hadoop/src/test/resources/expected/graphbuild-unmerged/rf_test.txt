((2,1)	[(2,2)]	[]	[(1,2)]	[]	ACCGT)	(null)
((2,2)	[(2,3)]	[]	[(1,1)]	[(2,1)]	CCGTG)	(null)
((2,3)	[(2,4)]	[]	[]	[(2,2)]	CGTGG)	(null)
((2,4)	[]	[]	[]	[(2,3)]	GTGGT)	(null)
((1,1)	[(1,2)]	[]	[(2,2)]	[]	ACGGT)	(null)
((1,2)	[(1,3)]	[]	[(2,1)]	[(1,1)]	CGGTG)	(null)
((1,3)	[(1,4)]	[]	[]	[(1,2)]	GGTGT)	(null)
((1,4)	[]	[]	[]	[(1,3)]	GTGTA)	(null)
